app/src/main/java/com/appunite/loudius/ui/login/LoginScreen.kt

```diff
@@ -81,7 +83,9 @@ fun LoginScreenStateless(
     ) {
         LoginImage()
         LoudiusOutlinedButton(
-            modifier = Modifier.fillMaxWidth(),
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(horizontal = 46.dp),
```

Comment:
Question: In the previous implementation, padding was added to `LoudiusOutlinedButton` if `style` was `LoudiusOutlinedButtonStyle.Large`. We don't have this condition here, is it ok?

Reply:
We don't use Large Buttons anywhere besides previews, so the extra margin was probably omitted.

Reply:
Hmm, I removed the padding from the `LoudiusOutlinedButton` cause it had external padding. Such components shouldn't have a margin added to them. So it was added by mistake. 
And here in `LoginScreen` the margin is added so that the UI stays the same. 
I don't know what you mean about the condition. I believe it is the same condition, we have a large button, and we have the same padding as before, but added here. 🤔

Reply:
This bug/mistake came up in one of my previous PR, but I've decided to fix it later. I could have said it before. 👍